server/monitor-types/tls.js

```diff
@@ -0,0 +1,266 @@
+const { MonitorType } = require("./monitor-type");
+const { log, UP } = require("../../src/util");
+const net = require("net");
+const tls = require("tls");
+
+class TlsMonitorType extends MonitorType {
+
+    name = "port-tls";
```

Comment:
think the naming here is a bit confusing.
I think more fitting might be `tcp-keyword` and ``TCPKeywordMonitorType`.
What do you think?

Reply:
I had originally just called it "TLS" but Louis suggested naming it "TCP Port (TLS)" at the UI level.

Since the "TCP Port" monitor uses `port` as the key, I figured `port-tls` was the most consistent choice.

The main discerning feature of the new monitor is really the TLS part and not so much the keyword. That was just an addition I found useful and, frankly, I would find it equally useful for the "TCP Port" monitor.

I'm open to renaming stuff but I'd definitely suggest for the new name to somehow revolve around the word "TLS" rather than "keyword". Otherwise things will get confusing if anyone adds keyword support to the existing "TCP Port" monitor.

Reply:
This is moot once we merge the two monitors.